src/seismometer/data/loader/prediction.py

```diff
@@ -0,0 +1,120 @@
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", category=FutureWarning)
+            present_columns = set(pq.ParquetDataset(config.prediction_path, use_legacy_dataset=False).schema.names)
+
+        if config.target in present_columns:
+            desired_columns.add(config.target)
+
+        actual_columns = desired_columns & present_columns
+        _log_column_mismatch(actual_columns, desired_columns, present_columns)
+
+        dataframe = pd.read_parquet(config.prediction_path, columns=actual_columns)
+    else:
+        dataframe = pd.read_parquet(config.prediction_path)
+
+    dataframe = _rename_targets(config, dataframe)
+
+    return dataframe.sort_index(axis=1)  # parquet can shuffle column order
+
+
+def _log_column_mismatch(actual_columns: list[str], desired_columns: list[str], present_columns: list[str]) -> None:
+    """Logs warnings if the actual columns and desired columns are a mismatch."""
+    if len(actual_columns) == len(desired_columns):
+        return
+    logger.warning(
+        "Not all requested columns are present. " + f"Missing columns are {', '.join(desired_columns-present_columns)}"
+    )
+    logger.debug(f"Requested columns are {', '.join(desired_columns)}")
+    logger.debug(f"Columns present are {', '.join(present_columns)}")
+
+
+def _rename_targets(config: ConfigProvider, dataframe: pd.DataFrame) -> pd.DataFrame:
+    """Renames the target column if alread in dataframe, to match what a event merge would produce."""
+    if config.target in dataframe:
+        target_value = pdh.event_value(config.target)
+        logger.debug(f"Using existing column in predictions dataframe as target: {config.target} -> {target_value}")
+        dataframe = dataframe.rename({config.target: target_value}, axis=1)
+    return dataframe
+
+
+# post loaders
+def assumed_types(config: ConfigProvider, dataframe: pd.DataFrame) -> pd.DataFrame:
+    """
+    Convert the loaded predictions dataframe to the expected types.
+
+    Scope is currently restricted to time and output columns as parquet is expected to include datatypes.
+
+    Parameters
+    ----------
+    config : ConfigProvider
+        The loaded configuration object.
+    dataframe : pd.DataFrame
+        The loaded predictions dataframe.
+
+    Returns
+    -------
+    pd.DataFrame
+        The predictions dataframe with adjusted types.
+    """
+    dataframe = _infer_datetime(dataframe)
+
+    # datetime precisions don't play nicely - fix to pands default
+    pred_times = dataframe.select_dtypes(include="datetime").columns
+    dataframe[pred_times] = dataframe[pred_times].astype({col: "<M8[ns]" for col in pred_times})
+
+    # Expand this to robust score prep
+    for score in config.output_list:
+        if score not in dataframe:
+            continue
+        if 50 < dataframe[score].max() <= 100:  # Assume out of 100, readjust
+            dataframe[score] /= 100
```

Comment:
Not this PR, but this was copied from seismogram.py and it feels brittle, since many of our scores are expected to trigger at low values due to the low prevalence.

Reply:
I halved the minimum as a bandaid "improvement".  
I think this will need to be reconsidered once we have more robust support for other ranges of predictions (my first thought is we'll need a flag or enum in configuration)